modules/percolator/src/test/java/org/elasticsearch/percolator/CandidateQueryTests.java

```diff
@@ -19,6 +19,23 @@
 
 package org.elasticsearch.percolator;
 
+import static org.elasticsearch.common.network.InetAddresses.forString;
```

Comment:
Can you revert the import changes? They'll get changed back next time somebody with different settings edits the file and it just adds noise.

Reply:
I am trying to figure out what is going on. I think I configured the formatter based on our formatting file, and since then imports order get automatically adjusted. Not loving it either in projects where we don't yet auto-format files.